src/features/jobs/JobsView.vue

```diff
@@ -75,6 +96,12 @@ export default {
   },
   // Map the state from store/index.js
   computed: mapState(["jobs", "jobsState"]),
+  methods: {
+    deleteJob(jobId) {
+      API.deleteJobById(jobId);
+      window.location.reload();
+    },
```

Comment:
Ein `reload()` ist keine gute Idee. Wir arbeiten in einer Single Page Application (SPA) und hier musst du einfach nur die Daten im Hintergrund laden:

```js
  methods: {
    async deleteJob(jobId) {
      await API.deleteJobById(jobId);
      // reload jobs
      this.$store.dispatch("getJobs");
    },
    // ...
  }
```
Das `await` vor dem löschen ist wichtig, weil wir erst dann die neuen Daten von der API laden dürfen.

Reply:
Ich habe den Vorschlag eingebaut, leider behebt das noch immer nicht das unten beschriebene Problem :(